src/app/feedback/feedback.directive.ts

```diff
@@ -20,13 +20,15 @@ export class Feedback {
   source: string;
   url: string;
   messages: Array<Message>;
+  Resource_title: string;
```

Comment:
I think we need to define it in such a way that it supports to all. Suppose for course feedback , it will not valid. So , could you please check it once ?